integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWDTModelNoServer.java

```diff
@@ -0,0 +1,356 @@
+
+    // check there is also a pod with name domainuid-myadmin running
+    checkPodReadyAndServiceExists(domainUid + "-myadmin", domainUid, domainNamespace);
+
+    //check and wait for the application to be accessible in all server pods
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is no AdminServerName and in Server section the server 'AdminServer' defined.
+   * Verify that AdminServerName is set to 'AdminServer' and the admin server pod is created.
+   */
+  @Test
+  @DisplayName("Test in the WDT model file there is no AdminServerName and in Server section AdminServer defined")
+  void testWdtModelNoAdminServerNameWithAdminServerInServer() {
+
+    String wdtModelFile = "model-noadminservername-hasadminserver.yaml";
+    imageName = createAndVerifyDomainImage(wdtModelFile);
+
+    createMiiDomainAndVerify(domainNamespace, domainUid,
+        imageName, adminServerPodName, managedServerPrefix, replicaCount);
+
+    //check and wait for the application to be accessible in all server pods
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is no AdminServerName and in Server section the server 'adminserver' defined.
```

Comment:
Explicitly mention that :  the server name is set to  adminserver ( all lowercase) as follow

 Test in the WDT model file there is no AdminServerName and in Server section the server 'adminserver'  ( all lowercase) defined.

Reply:
updated the javadoc